src/blocks/inline-notice/styling.js

```diff
@@ -109,45 +129,61 @@ function styling( attributes, clientId, name, deviceType ) {
 
 	if ( noticeDismiss ) {
 		if ( 'left' === noticeAlignment || 'center' === noticeAlignment ) {
-			rPadding = titleRightPadding;
-			lPadding = titleLeftPadding;
-			lPaddingMobile = titleLeftPaddingMobile;
-			rPaddingMobile = titleRightPaddingMobile;
-			lPaddingTablet = titleLeftPaddingTablet;
-			rPaddingTablet = titleRightPaddingTablet;
+			rPadding = titlePaddingRight;
+			lPadding = titlePaddingLeft;
+			lPaddingMobile = titlePaddingLeftMobile;
+			rPaddingMobile = titlePaddingRightMobile;
+			lPaddingTablet = titlePaddingLeftTablet;
+			rPaddingTablet = titlePaddingRightTablet;
 		} else {
-			lPadding = titleLeftPadding;
-			rPadding = titleRightPadding;
-			lPaddingMobile = titleLeftPaddingMobile;
-			rPaddingMobile = titleRightPaddingMobile;
-			lPaddingTablet = titleLeftPaddingTablet;
-			rPaddingTablet = titleRightPaddingTablet;
+			lPadding = titlePaddingLeft;
+			rPadding = titlePaddingRight;
+			lPaddingMobile = titlePaddingLeftMobile;
+			rPaddingMobile = titlePaddingRightMobile;
+			lPaddingTablet = titlePaddingLeftTablet;
+			rPaddingTablet = titlePaddingRightTablet;
 		}
 	} else {
-		lPadding = titleLeftPadding;
-		rPadding = titleRightPadding;
-		lPaddingMobile = titleLeftPaddingMobile;
-		rPaddingMobile = titleRightPaddingMobile;
-		lPaddingTablet = titleLeftPaddingTablet;
-		rPaddingTablet = titleRightPaddingTablet;
+		lPadding = titlePaddingLeft;
+		rPadding = titlePaddingRight;
+		lPaddingMobile = titlePaddingLeftMobile;
+		rPaddingMobile = titlePaddingRightMobile;
+		lPaddingTablet = titlePaddingLeftTablet;
+		rPaddingTablet = titlePaddingRightTablet;
 	}
 
 	// Setting the Icon Position Defaults
-	const posTopTab = ! isNaN( titleTopPaddingTablet ) ? titleTopPaddingTablet : titleTopPadding;
-	const posLeftTab = ! isNaN( titleLeftPaddingTablet ) ? titleLeftPaddingTablet : titleLeftPadding;
-	const posRightTab = ! isNaN( titleRightPaddingTablet ) ? titleRightPaddingTablet : titleRightPadding;
+	const posTopTab = ! isNaN( titlePaddingTopTablet ) ? titlePaddingTopTablet : titlePaddingTop;
```

Comment:
Should we follow the more precise number check as discussed over call for this as well?

```suggestion
	const posTopTab = ( 'number' === typeof titlePaddingTopTablet ) ? titlePaddingTopTablet : titlePaddingTop;
```

Same for all other places below.

Reply:
Changes done.